### PR TITLE
Merge level graphs

### PR DESCRIPTION
**Pull request by key6oardWarrior**
@trilinos/<teamName>

## Motivation
We need a python tool to combine files into a nested dot file. Issue: https://github.com/trilinos/Trilinos/issues/9534

## Testing
N/A

### CODE REVIEW

**Comment by key6oardWarrior** [comment]
@cgcgcg @csiefer2 Are susposed to be reviewers, but Github did not let me add them for some reason. There was never an option to add anyone as a reviewer.

**Review by cgcgcg — APPROVED** [review]
Kick off the autotester, but don't merge yet

**Comment by cgcgcg** [comment]
@trilinos/muelu

File: packages/muelu/utils/MergeLevelGraphs/mergeLevelGraphs.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+			while os.path.exists(imageFile):
+				ans = input(f"Are you sure you want to overwrite this file: {imageFile}\n Enter \"y\" to pick a different file name: ")
+				if ans.strip().lower() == "y":
+					ans = input("Enter file path: ")
+					imageFile = ans
+				else:
+					break
+		else:
+			graphFile = os.path.join(PATH, "output.dot")
+			imageFile = os.path.join(PATH, "output.png")
```
Comment (cgcgcg) [inline packages/muelu/utils/MergeLevelGraphs/mergeLevelGraphs.py:282]: The output is saved to the location of the script. That seems non-standard and leads to unintuitive behavior. Rather use the location from where the script is called.

**Review by cgcgcg — CHANGES_REQUESTED** [review]
The script fails when called on dot files that are in a different folder, since the file name containing a relative path is used for the cluster names. In the output png, the clusters have no labels.

**Review by cgcgcg — APPROVED** [review]

**Pull request merged**